css-canvas-medical/styles.css

```diff
@@ -0,0 +1,247 @@
+
+body {
+  font-family: sans-serif;
+  font-size: 12px;
+  border: 4px solid red;
+}
+
+p,
+li,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+h1 > p {
+  width: 80%;
+  line-height: 2rem;
+  margin-top: 4rem;
+}
+
+h2 {
+  padding-top: 0;
+  margin-top: 0;
+  font-weight: bold;
+}
+
+#big-container {
+  height: auto;
+  width: 1200px;
+  align-items: center;
+  margin-left: 10%;
+  margin-right: 10%;
+  padding-top: 2%;
+}
+
+#container3 > .physician2 {
+  margin: 0;
+  padding: 0;
+}
+
+ul > a {
+  text-decoration: none;
+  color: gray;
+  font-weight: bold;
+}
+
+p > a {
+  text-decoration: none;
+  color: white;
+}
+
+#container,
+#container2 {
+  width: 100%;
+}
+
+.header-items {
+  padding-top: 1%;
+  border-bottom: 1px solid grey;
+  font-weight: lighter;
+}
+
+.header-logo {
+  position: relative;
+  width: 155px;
+  margin-top: 0;
+  margin-left: 20px;
+}
+
+.wrapper {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  margin-left: 45%;
```

Comment:
😱 😱 😱 😱 😱 

That is A LOT of margin! Can this be done differently, in a way that doesn't require such manual movements?